docs/cloud-workload-security/agent_expressions.md

```diff
@@ -210,6 +210,11 @@ The *file.rights* attribute can now be used in addition to *file.mode*. *file.mo
 | `process.ancestors.file.mount_id` | int | Mount ID of the file |  |
 | `process.ancestors.file.name` | string | File's basename |  |
 | `process.ancestors.file.name.length` | int | Length of 'process.ancestors.file.name' string |  |
+| `process.ancestors.file.package.major` | int | Major version of the package provided this file |  |
+| `process.ancestors.file.package.minor` | int | Minor version of the package provided this file |  |
+| `process.ancestors.file.package.name` | string | Name of the package provided this file |  |
+| `process.ancestors.file.package.patch` | int | Patch version of the package provided this file |  |
+| `process.ancestors.file.package.version` | string | Full version of the package provided this file |  |
```

Comment:
Here and throughout this file: Should this say "... of the package provided in this file" or maybe "provided by this file"? There's a word missing.